src/utils/events/event-manager.js

```diff
@@ -76,38 +76,46 @@ export default class EventManager {
    */
   on(event, handler) {
```

Comment:
`.on()/.off()` is a very common idiom for adding event handlers. [Node](https://nodejs.org/api/events.html#events_emitter_on_eventname_listener) and [jQuery](http://api.jquery.com/on/) both use it, and it's less verbose than e.g. `addEventListener`. I think it's fine as-is but if you feel strongly about it we can change it.